pkg/controller/garbagecollector/absent_owner_cache.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package garbagecollector
+
+import (
+	"sync"
+
+	"github.com/golang/groupcache/lru"
+	"k8s.io/kubernetes/pkg/types"
+)
+
+// AbsentOwnerCache caches the owner that does not exist according to API server's responses.
+type AbsentOwnerCache struct {
```

Comment:
I think names should be about what the object does, not about what you're
going to use it for. So I think just "UIDCache" is better.

On Mon, Aug 22, 2016 at 6:36 PM, Chao Xu <notifications@github.com> wrote:

> In pkg/controller/garbagecollector/absent_owner_cache.go
> <https://github.com/kubernetes/kubernetes/pull/31167#discussion_r75790485>
> :
>
> > +WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +See the License for the specific language governing permissions and
> > +limitations under the License.
> > +*/
> > +
> > +package garbagecollector
> > +
> > +import (
> > +	"sync"
> > +
> > +	"github.com/golang/groupcache/lru"
> > +	"k8s.io/kubernetes/pkg/types"
> > +)
> > +
> > +// AbsentOwnerCache caches the owner that does not exist according to API server's responses.
> > +type AbsentOwnerCache struct {
>
> How about NonExistentUIDCache?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/31167/files/0f45f5c02d3710efc6362d5d440a3d18b3a4835f#r75790485>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAnglqxVSbW8MzOklpCcbk4tge1YedXQks5qik6egaJpZM4JqWZd>
> .
>